packages/trader/src/AppV2/Containers/Trade/trade.tsx

```diff
@@ -88,17 +88,19 @@ const Trade = observer(() => {
                         <TradeParametersContainer>
                             <TradeParameters />
                         </TradeParametersContainer>
-                        <section className='trade__chart' style={{ height: dynamic_chart_height }} ref={chart_ref}>
-                            <TradeChart />
-                        </section>
+                        <div className='trade__chart-tooltip'>
+                            <section className='trade__chart' style={{ height: dynamic_chart_height }} ref={chart_ref}>
+                                <TradeChart />
+                            </section>
+                        </div>
                         {isAccumulatorContract(contract_type) && <AccumulatorStats />}
                     </div>
-                    <div className='trade__bottom'>
+                    <span className='trade__bottom'>
```

Comment:
Does it complain, that in span we have block tags?

Reply:
I mean console errors?

Reply:
I have made this change